deployments/azure-kubernetes-service-helm.yml

```diff
@@ -96,7 +112,30 @@ jobs:
         container-registry-url: ${{ env.AZURE_CONTAINER_REGISTRY }}.azurecr.io
         container-registry-username: ${{ steps.get-acr-creds.outputs.username }}
         container-registry-password: ${{ steps.get-acr-creds.outputs.password }}
-        secret-name: ${{ env.IMAGE_PULL_SECRET_NAME }}
+        secret-name: ${{ env.IMAGE_PULL_SECRET_NAME }} 
+
+  deploy:
+    permissions:
+      actions: read
```

Comment:
here as well

Reply:
actions: read seems to be needed for the deploy stage. Still unsure why. The rest should be able to go. Good catch!
https://github.com/actions/starter-workflows/pull/1403